pkg/server/webhook.go

```diff
@@ -419,12 +427,11 @@ func createPatch(pod *corev1.Pod, inj *config.InjectionConfig, annotations map[s
 }
 
 // main mutation process
-func (whsvr *WebhookServer) mutate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
-	req := ar.Request
+func (whsvr *WebhookServer) mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
```

Comment:
i think it makes more sense to expose an AdmissionRequest/AdmissionResponse, instead of AdmissionReview/AdmissionResponse. Just tweaking up the interface here